### DISCUSSION TITLE
signal-desktop: program fails on startup

### DISCUSSION THREAD

**Discussion by rieje** [netblue30/firejail#5725] [Q&A] [answered]
Anyone have a guess why signal-desktop fails on system startup? I run it as a systemd user service. It pretty much never starts on startup and I need to restart the service and then it will launch. On the rare occasion I think it might randomly crash but I'm not sure. If I don't run it without firejail it works fine without fail. Here's [systemd service status](https://0x0.st/HiAg.txt). I'm on Wayland (Sway).

**Comment by kmk3** [comment, marked as answer]
Moved to an issue:

* <https://github.com/netblue30/firejail/issues/5726>
